packages/lesmis/src/components/EntityVitalsItem.js

```diff
@@ -0,0 +1,122 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ *
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import MuiContainer from '@material-ui/core/Container';
+import {
+  PinDrop,
+  People,
+  School,
+  LocalLibrary,
+  Assignment,
+  Map,
+  LocationOn,
+} from '@material-ui/icons';
+import Typography from '@material-ui/core/Typography';
+
+const Wrapper = styled.section`
+  padding-top: 0.5rem;
+  padding-left: 0;
+  padding-right: 0;
+  padding-bottom: 0;
+`;
+
+const Container = styled(MuiContainer)`
+  display: flex;
+  flex-direction: row;
+  width: 30%;
+  margin-left: 0;
+  margin-right: 0;
+`;
```

Comment:
```suggestion
const Container = styled(MuiContainer)`
     .MuiSvgIcon-root {
         padding-top: 0.5rem;
         font-size: 2.5rem;
         color: ${props => props.theme.palette.text.secondary};
       }
`;
```

```suggestion

```

The `MuiContainer` is designed as a top level container as it has things like a page gutter and max width 1200px etc. I would also leave the spacing of 30% to the user of this component which will make it more flexible. So then we are just left with the display: flex and flex-direction: row and there is a component for that in the Layout.js file.

Reply:
I've swapped these for the actual icons so this formatting is no longer necessary, I've cleared it all out